easybuild/easyblocks/g/gromacs.py

```diff
@@ -493,3 +563,118 @@ def sanity_check_step(self):
+            # the MPI variant. So we need to be able to specify the
+            # install target depending on variant.
+            self.cfg['install_cmd'] = 'make'
+
+            # Use the fact that for older versions we just need to
+            # build and install mdrun for the MPI part
+            build_opts = {
+                'nompi': '',
+                'mpi': 'mdrun'
+            }
+            install_opts = {
+                'nompi': 'install',
+                'mpi': 'install-mdrun'
+            }
+        else:
+            build_opts = {
+                'nompi': '',
+                'mpi': ''
+            }
+            install_opts = {
+                'nompi': 'install',
+                'mpi': 'install'
+            }
+
+        precisions = ['single']
+        if self.cfg.get('double_precision') is None or self.cfg.get('double_precision'):
+            precisions.append('double')
+
+        mpitypes = ['nompi']
+        if self.toolchain.options.get('usempi', None):
+            mpitypes.append('mpi')
+
+        # We need to count the numb er of variations to build.
+        self.variants_to_build = 0
+        versions_built = []
+        # Handle the different variants
+        for precicion in precisions:
+            for mpitype in mpitypes:
+                versions_built.append('%s precision %s' % (precicion, mpitype))
+                var_confopts = []
+                var_buildopts = []
+                var_installopts = []
+
+                var_confopts.append(mpi_type_opts[mpitype])
+                var_confopts.append(prec_opts[precicion])
+                if LooseVersion(self.version) < LooseVersion('4.6'):
+                    suffix = ''
+                    if mpitype == 'mpi':
+                        suffix = "--program-suffix={0}".format(self.cfg.get('mpisuffix'))
+                        if precicion == 'double':
+                            suffix += '_d'
+                    var_confopts.append(suffix)
+
+                var_buildopts.append(build_opts[mpitype])
+                var_installopts.append(install_opts[mpitype])
+
+                self.cfg.update('configopts', [' '.join(var_confopts) + ' ' + common_config_opts])
+                self.cfg.update('buildopts', [' '.join(var_buildopts) + ' ' + common_build_opts])
+                self.cfg.update('installopts', [' '.join(var_installopts) + ' ' + common_install_opts])
+        self.variants_to_build = len(self.cfg['configopts'])
+
+        self.log.debug("List of configure options to iterate over: %s", self.cfg['configopts'])
+        self.log.info("Building these versions of GROMACS: %s", ', '.join(versions_built))
+        return super(EB_GROMACS, self).run_all_steps(*args, **kwargs)
+
+        self.cfg['install_cmd'] = self.save_install_cmd
+        self.cfg['build_cmd'] = self.save_build_cmd
+
+        self.log.info("A full regression test suite is available from the GROMACS web site")
```

Comment:
This log message is useful without a URL? :)

Reply:
Yeah, but perhaps one could pull homepage from the easyconfig easily?

Reply:
Why not mention a direct URL in the log message?

Reply:
My question above was if I can pick the homepage value from the easyconfig easily and use that in the log message. More future proof

Reply:
Sure: `self.cfg['homepage']` (just like any other easyconfig parameter)

Reply:
Done